modules/advanced-icon/includes/frontend.css.php

```diff
@@ -0,0 +1,179 @@
+
+foreach ( $settings->icons as $i => $icon ) :
+
+		$imageicon_array = array(
+
+			/* General Section */
+			'image_type'              => $icon->image_type,
+
+			/* Icon Basics */
+			'icon'                    => $icon->icon,
+			'icon_size'               => $settings->size,
+			'icon_align'              => $settings->align,
+
+			/* Image Basics */
+			'photo_source'            => 'library',
+			'photo'                   => $icon->photo,
+			'photo_url'               => '',
+			'img_size'                => ( 'custom' === $settings->icoimage_style || 'simple' === $settings->icoimage_style ) ? $settings->size : ( $settings->size * 2 ),
+			'img_align'               => $settings->align,
+			'photo_src'               => ( isset( $icon->photo_src ) ) ? $icon->photo_src : '',
+
+			/* Icon Style */
+			'icon_style'              => $settings->icoimage_style,
+			'icon_bg_size'            => ( $settings->bg_size * 2 ),
+			'icon_border_style'       => $settings->border_style,
+			'icon_border_width'       => $settings->border_width,
+			'icon_bg_border_radius'   => $settings->bg_border_radius,
+
+			/* Image Style */
+			'image_style'             => $settings->icoimage_style,
+			'img_bg_size'             => $settings->bg_size,
+			'img_border_style'        => $settings->border_style,
+			'img_border_width'        => $settings->border_width,
+			'img_bg_border_radius'    => $settings->bg_border_radius,
+
+			/* Preset Color variable new */
+			'icon_color_preset'       => $settings->color_preset,
+
+			/* Icon Colors */
+			'icon_color'              => ( ! empty( $icon->icocolor ) ) ? $icon->icocolor : $settings->color,
+			'icon_hover_color'        => ( ! empty( $icon->icohover_color ) ) ? $icon->icohover_color : $settings->hover_color,
+			'icon_bg_color'           => ( ! empty( $icon->bg_color ) ) ? $icon->bg_color : $settings->bg_color,
+			'icon_bg_hover_color'     => ( ! empty( $icon->bg_hover_color ) ) ? $icon->bg_hover_color : $settings->bg_hover_color,
+			'icon_border_color'       => ( ! empty( $icon->border_color ) ) ? $icon->border_color : $settings->border_color,
+			'icon_border_hover_color' => ( ! empty( $icon->border_hover_color ) ) ? $icon->border_hover_color : $settings->border_hover_color,
+			'icon_three_d'            => $settings->three_d,
+
+			/* Image Colors */
+			'img_bg_color'            => ( ! empty( $icon->bg_color ) ) ? $icon->bg_color : $settings->bg_color,
+			'img_bg_hover_color'      => ( ! empty( $icon->bg_hover_color ) ) ? $icon->bg_hover_color : $settings->bg_hover_color,
+			'img_border_color'        => ( ! empty( $icon->border_color ) ) ? $icon->border_color : $settings->border_color,
+			'img_border_hover_color'  => ( ! empty( $icon->border_hover_color ) ) ? $icon->border_hover_color : $settings->border_hover_color,
+		);
+
+
+	FLBuilder::render_module_css( 'image-icon', $id . ' .adv-icon-' . $icon_count, $imageicon_array );
+
+	if ( isset( $settings->responsive_align ) ) {
```

Comment:
Do we need this responsive code in loop?

Reply:
Yes, as here we have set desktop alignment in a loop for as many icons we create through module hence We have to add this responsive setting also in the Loop.